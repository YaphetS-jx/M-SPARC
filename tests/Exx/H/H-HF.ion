ATOM_TYPE: H
PSEUDO_POT: H.pot
N_TYPE_ATOM: 1
COORD:
10 10 10