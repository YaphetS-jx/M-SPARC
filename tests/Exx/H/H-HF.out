***************************************************************************
*                      M-SPARC v1.0.0 (Oct 07, 2020)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 05-Jan-2021  Start time: 19:41:20                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 20.000000 20.000000 20.000000 
FD_GRID: 67 67 67
FD_ORDER: 12
BC: D D D
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.000999993
CHEB_DEGREE: 26
NSTATES: 2
EXCHANGE_CORRELATION: HF
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 1
PRINT_DENSITY: 0
MAXIT_FOCK: 100
TOL_FOCK: 1.00E-06
HYB_MIXING: 1.00
EXX_METHOD: FOURIER_SPACE
OUTPUT_FILE: ../tests/Exx/H/H-HF.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.298507 (Bohr)
Output printed to                  :  ../tests/Exx/H/H-HF.out
Total number of atom types         :  1
Total number of atoms              :  1
Total number of electrons          :  1
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  H.pot
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.87 6.87 6.87
Number of atoms of type 1          :  1
Estimated total memory usage       :  1.17 GB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -4.5579798393E-01        3.299E-01        6.405
2            -4.5970009241E-01        1.208E-01        7.454
3            -4.5994112464E-01        9.731E-02        7.439
4            -4.5995877258E-01        7.401E-02        6.365
5            -4.5996084517E-01        2.361E-02        5.744
6            -4.5996107524E-01        2.678E-03        5.384
7            -4.5996109194E-01        1.247E-03        5.070
8            -4.5996109322E-01        4.104E-04        4.615
9            -4.5996109283E-01        2.178E-04        3.765
10           -4.5996109265E-01        1.061E-04        3.306
11           -4.5996109202E-01        3.096E-05        3.118
12           -4.5996109403E-01        7.840E-06        2.609
13           -4.5996109462E-01        7.274E-06        2.277
14           -4.5996109289E-01        4.030E-06        1.993
15           -4.5996107918E-01        1.076E-06        1.899
16           -4.5996108016E-01        1.264E-07        1.763
Total number of SCF: 16    
No.1 Exx outer loop:
1            -5.0148647965E-01        9.710E-02        26.588
2            -5.0184332401E-01        5.600E-02        10.147
3            -5.0201810567E-01        1.735E-03        10.065
4            -5.0201880715E-01        7.447E-04        9.727
5            -5.0201882198E-01        3.286E-04        8.899
6            -5.0201882250E-01        2.779E-04        7.829
7            -5.0201882431E-01        3.815E-05        7.669
8            -5.0201882329E-01        3.092E-05        8.086
9            -5.0201882126E-01        1.584E-05        6.634
10           -5.0201882454E-01        4.739E-06        6.646
11           -5.0201882586E-01        7.873E-07        6.390
Total number of SCF: 11    
 Eex = 0.13877031
 Etot = -0.36324852
No.2 Exx outer loop:
1            -4.9266866067E-01        3.577E-02        25.904
2            -4.9271177141E-01        2.068E-02        9.529
3            -4.9273279576E-01        1.215E-04        10.292
4            -4.9273285501E-01        1.029E-04        8.884
5            -4.9273285491E-01        5.470E-05        8.212
6            -4.9273285464E-01        4.138E-05        7.401
7            -4.9273285495E-01        1.969E-05        6.031
8            -4.9273285702E-01        1.374E-05        6.822
9            -4.9273285262E-01        9.520E-06        5.952
10           -4.9273285551E-01        2.833E-06        6.064
11           -4.9273285146E-01        2.284E-06        6.233
12           -4.9273285355E-01        9.524E-07        5.728
Total number of SCF: 12    
 Eex = 0.13201083
 Etot = -0.36072202
No.3 Exx outer loop:
1            -4.8920626456E-01        1.315E-02        26.197
2            -4.8921171761E-01        7.629E-03        9.143
3            -4.8921437324E-01        8.314E-05        8.655
4            -4.8921438061E-01        2.075E-05        8.617
5            -4.8921437974E-01        2.381E-05        6.858
6            -4.8921437944E-01        1.840E-05        6.671
7            -4.8921437895E-01        6.171E-06        6.085
8            -4.8921437861E-01        1.292E-05        6.549
9            -4.8921438036E-01        5.315E-06        6.681
10           -4.8921438348E-01        2.073E-06        6.330
11           -4.8921438525E-01        3.255E-07        5.885
Total number of SCF: 11    
 Eex = 0.12955626
 Etot = -0.35965813
No.4 Exx outer loop:
1            -4.8791349603E-01        4.818E-03        26.039
2            -4.8791420270E-01        2.802E-03        8.807
3            -4.8791454557E-01        4.636E-05        8.366
4            -4.8791454696E-01        3.559E-06        8.359
5            -4.8791454527E-01        3.190E-06        6.926
6            -4.8791454778E-01        9.623E-07        6.269
Total number of SCF: 6     
 Eex = 0.12866232
 Etot = -0.35925222
No.5 Exx outer loop:
1            -4.8743826910E-01        1.761E-03        26.349
2            -4.8743836472E-01        1.027E-03        8.374
3            -4.8743840947E-01        9.013E-06        7.985
4            -4.8743840521E-01        4.803E-06        8.174
5            -4.8743840339E-01        7.656E-06        6.381
6            -4.8743841054E-01        2.606E-06        6.004
7            -4.8743840880E-01        6.972E-07        5.901
Total number of SCF: 7     
 Eex = 0.12833653
 Etot = -0.35910188
No.6 Exx outer loop:
1            -4.8726450834E-01        6.436E-04        26.200
2            -4.8726452191E-01        3.755E-04        8.274
3            -4.8726452806E-01        1.743E-06        7.985
4            -4.8726452988E-01        5.211E-07        7.996
Total number of SCF: 4     
 Eex = 0.12821773
 Etot = -0.35904680
No.7 Exx outer loop:
1            -4.8720101630E-01        2.348E-04        26.142
2            -4.8720102019E-01        1.372E-04        7.864
3            -4.8720102091E-01        5.802E-07        7.724
Total number of SCF: 3     
 Eex = 0.12817439
 Etot = -0.35902663
No.8 Exx outer loop:
1            -4.8717787602E-01        8.575E-05        27.118
2            -4.8717787533E-01        5.046E-05        7.931
3            -4.8717787705E-01        5.159E-07        7.007
Total number of SCF: 3     
 Eex = 0.12815858
 Etot = -0.35901930
No.9 Exx outer loop:
1            -4.8716940974E-01        3.099E-05        25.865
2            -4.8716940731E-01        1.842E-05        7.674
3            -4.8716940847E-01        4.949E-07        6.325
Total number of SCF: 3     
 Eex = 0.12815281
 Etot = -0.35901660
No.10 Exx outer loop:
1            -4.8716633500E-01        1.096E-05        25.726
2            -4.8716633166E-01        6.588E-06        6.971
3            -4.8716633251E-01        6.254E-07        5.914
Total number of SCF: 3     
 Eex = 0.12815072
 Etot = -0.35901562
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -3.5901561545E-01 (Ha/atom)
Total free energy                  : -3.5901561545E-01 (Ha)
Band structure energy              : -2.2950886850E-01 (Ha)
Exchange correlation energy        : -0.0000000000E+00 (Ha)
Self and correction energy         : -1.6578888281E+00 (Ha)
Entropy*kb*T                       : -1.3862850069E-03 (Ha)
Fermi level                        : -2.2950887506E-01 (Ha)
Average force                      :  0.0000000000E+00 (Ha/Bohr)
Maximum force                      :  0.0000000000E+00 (Ha/Bohr)
Time for force calculation         :  1.617 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  819.034 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
